compiler/crates/relay-compiler/src/config.rs

```diff
@@ -696,15 +702,20 @@ struct MultiProjectConfigFile {
     feature_flags: FeatureFlags,
 
     /// Watchman saved state config.
+    #[schemars(skip)] // TODO: Fix me
```

Comment:
This one's tricky because `ScmAwareClockData` and its serialization/deserialization is defined in its own crate which we don't own. We probably need to define our own wrapper around `ScmAwareClockData` that replicates its fields as suggested in https://graham.cool/schemars/examples/5-remote_derive/